src/components/shared/Address/InvisibleCopyableMaskedAddress/InvisibleCopyableMaskedAddress.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Address } from '~types';
+
+import MaskedAddress from '~shared/MaskedAddress';
+import InvisibleCopyableAddress from '~shared/InvisibleCopyableAddress';
+
+import styles from './InvisibleCopyableMaskedAddress.css';
+
+const displayName = 'InvisibleCopyableMaskedAddress';
+
+interface InvisibleCopyableMaskedAddressProps {
+  address: Address;
+}
+const InvisibleCopyableMaskedAddress = ({
+  address,
+}: InvisibleCopyableMaskedAddressProps) => (
+  <InvisibleCopyableAddress address={address}>
```

Comment:
Common pattern. Created separate issue https://github.com/JoinColony/colonyCDapp/issues/246 to handle implementing it everywhere since not directly relevant to this pr.